js/index.js

```diff
@@ -1 +1,105 @@
-// Your code goes here
+// CODE: //
+
+// ============== CONSTANTS ============ //
+
+const navLinks = document.querySelector('.nav');
```

Comment:
I recommend filling up this section with more selections, as it will allow you to select more parts of the page to manipulate with events.